test/SourceKit/DocSupport/doc_swift_module.swift.response

```diff
@@ -1529,7 +1529,7 @@ func genfoo<T1, T2>(x ix: T1, y iy: T2) where T1 : Prot, T2 : cake.C1, T1.Elemen
         key.entities: [
           {
             key.kind: source.lang.swift.decl.enumelement,
-            key.name: "case1",
+            key.name: "case1()",
```

Comment:
@benlangmuir Any idea what is causing SourceKit to serialize a null parameter list like this?

Reply:
SourceKit is just calling ValueDecl::getFullName.  This test is loading the decl from a Swift module though, so maybe check that?